src/System.Collections/tests/Generic/Dictionary/Dictionary.Generic.Tests.netcoreapp.cs

```diff
@@ -84,5 +84,52 @@ public void Dictionary_Generic_RemoveKey_DefaultKeyContainedInDictionary(int cou
         }
 
         #endregion
+
+        #region EnsureCapacity
+        
+        [Fact]
+        public void EnsureCapacity_NegativeCapacityRequested_Throws()
+        {
+            Dictionary<string, string> dictionary = new Dictionary<string, string>();
+            Assert.Throws<ArgumentOutOfRangeException>(() => dictionary.EnsureCapacity(-1));
+        }
+
+        [Theory]
+        [InlineData(0)]
+        [InlineData(1)]
+        [InlineData(3)]
+        public void EnsureCapacity_DefaultCapacityOnEmptyDictionary_ReturnsCapacityRequestedLargerOrEqual(int requestedCapacity)
+        {
+            Dictionary<string, string> dictionary = new Dictionary<string, string>();
+            Assert.True(requestedCapacity <= dictionary.EnsureCapacity(requestedCapacity));
+        }
+
+        [Theory]
+        [InlineData(1)]
+        [InlineData(3)]
+        public void EnsureCapacity_CapacityRequestedSmallerThanCurrentCapacity_CapacityUnchanged(int requestedCapacity)
+        {
+            Dictionary<string, string> dictionary = new Dictionary<string, string>();
+
```

Comment:
nit please be consistent in blank lines with above ... it's all about making it easier for the eye to scan the code and see irregularlities.

Reply:
My rationale for blank space is that I usually segment tests into 3 segments and I leave blank space between them: 
```
[assumption]

[action]

[assertion]
```
But I agree in this case which is a simple one-liner I better remove this extra blank lines

Reply:
that's fine just be consistent, the tests above don't have a line after you create the dictionary.